src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -152,8 +152,8 @@ module MOM_internal_tides
   integer :: En_restart_power !< A power factor of 2 by which to multiply the energy in restart [nondim]
   type(time_type), pointer :: Time => NULL() !< A pointer to the model's clock.
   character(len=200) :: inputdir !< directory to look for coastline angle file
-  real :: decay_rate    !< A constant rate at which internal tide energy is
-                        !! lost to the interior ocean internal wave field [T-1 ~> s-1].
+  real, allocatable, dimension(:,:,:,:) :: decay_rate_2d !< same as above but with a horizontal
+                        !! structure for each harmonic [T-1 ~> s-1].
```

Comment:
Now that `decay_rate` is no longer there, the description "same as above but ..." no longer works.  Consider changing the description to "A rate at which internal tide energy is lost to the interior ocean internal wave field, that can potentially vary with spatial position, vertical mode and harmonic frequency [T-1 ~> s-1]."